book_maker/translator/qwen_translator.py

```
@@ -0,0 +1,262 @@
+                if completion.choices[0].message.content:
+                    t_text = completion.choices[0].message.content.strip()
+                else:
+                    t_text = ""
+
+                # Save to context for translation memory
+                if self.context_flag and t_text:
+                    self.save_context(text, t_text)
+
+                break
+
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)